src/types.ts

```diff
@@ -0,0 +1 @@
+export type Nullable<T> = T | null;
```

Comment:
I think this is a meaningless type because `T | null` is way more clear at first glance.

Reply:
I created it because It looks pretty instead of using |. If team doesn't want it I can remove it. @wopehq/frontend